data/geany.glade

```diff
@@ -8739,13 +8739,31 @@
                   <placeholder/>
                 </child>
                 <child>
+                  <object class="GtkLabel" id="label26">
+                    <property name="visible">True</property>
+                    <property name="can_focus">False</property>
+                    <property name="xalign">0</property>
+                    <property name="label" translatable="yes">Note: To apply these settings to all currently open documents, use &lt;i&gt;Project-&gt;Apply Default Indentation&lt;/i&gt;.</property>
+                    <property name="use_markup">True</property>
+                    <property name="wrap">True</property>
+                    <property name="width_chars">64</property>
```

Comment:
This works around the [awfully buggy sizing code from GTK3's wrap-able labels](https://bugzilla.gnome.org/show_bug.cgi?id=657621) (the reason why we do have [`GeanyWrapLabel` on GTK3](/geany/geany/commit/d6fba7ac8244ebf077e76468a5072552fa02aa89), but it's not (currently) usable from Glade)